resources.tf

```diff
@@ -55,6 +55,13 @@ resource "local_file" "group_vars_slurm" {
           access: openstack_sharedfilesystem_share_access_v2.software_share_access_rw.access_key,
           access_to: openstack_sharedfilesystem_share_access_v2.software_share_access_rw.access_to
         }
+        data: {
```

Comment:
Oh cool! terraform scripts for adding a data mount! guessing this will be very useful during a hackathon

Reply:
Yeah. The elwazi hackathon repo I did is here: https://github.com/elwazi/elwazi-workshop-vm
If you're interested.